input/OFCB/test1/pb1.json

```diff
@@ -0,0 +1,93 @@
+{
+	"materials": {
+		"pwr_uox": {"attributes": {"recipe": true}, 
+			"constraints": [ 
```

Comment:
@Baaaaam :
Depending on your text editor, you can usually set a preference to never use tab. If you use vi, you can make this happen by adding the following line to your ~/.vimrc file:

:set expandtab